Add a pointer from "realising" to `nix log`. (#4876)
NixOS/nix: doc/manual/src/command-ref/nix-store.md

$ nix-build '<nixpkgs>' -A hello --check -K
```

> To see the stderr/stdout of a build:

Reviewer: ```suggestion
Use [`--read-log`](#operation---read-log) to show the stderr and stdout of a build:
```

```console
$ nix-store --read-log $(nix-instantiate ./test.nix)
```

> Use `nix log` to see the logs of a build.
> 

Reviewer: ````suggestion
Alternatively, use [`nix log`](./new-cli/nix3-nix-log.md) (experimental) to show the build logs:

```console
nix log $(nix-instantiate ./test.nix)
```
````

To be honest, I find it a bit hard to integrate it smoothly here, so I'd also be fine with removing the line completely: 

```suggestion
```
